cfme/base/__init__.py

```diff
@@ -257,6 +278,44 @@ def advanced_settings(self):
         """"GET zones/:id/settings api endpoint to query zone configuration"""
         return self.appliance.rest_api.get(self._api_settings_url)
 
+    @property
+    def current_string(self):
+        """Returns the string ' (current)' if the appliance serving the UI request is in this zone.
+        Used to generate the appropriate tree_path for navigating configuration accordion trees."""
+        return ' (current)' if self.description == self.appliance.server.zone.description else ''
+
+    @property
+    def title_string(self):
+        """Returns a string of the form:
+
+        "Zone Description" (current)
+        [or]
+        "Zone Description"
+
+        Used to generate the title.text for Zone-related views."""
+        return f"\"{self.description}\"{self.current_string}"
+
+    @property
+    def tree_path(self):
+        """Generate the tree path list for the settings tree in the configuration accordion
+
+        Returns:
+            list of path elements for tree navigation
+        """
+        path = [
+            self.region.settings_string,
+            "Zones",
+            f"Zone: {self.description}{self.current_string}"
```

Comment:
self.title_string here?

Reply:
title_string puts double quotes around the zone description, whereas the string in the tree path doesn't have them. I've updated title_string to accept an optional 'quote_str' string parameter so that the same method can be called. I had to remove the property decorator from title_string to make that work.